### PR TITLE
Update dependencies based on npm audit

### PR DESCRIPTION
**Pull request by alisman**
This PR updates some dependencies in accordance with npm audit

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -2037,7 +2039,7 @@
         },
         "string_decoder": {
           "version": "1.1.1",
-          "resolved": "https://registry.npmjs.org/string_decoder/-/string_decoder-1.1.1.tgz",
+          "resolved": "http://registry.npmjs.org/string_decoder/-/string_decoder-1.1.1.tgz",
```
Comment (inodb) [inline package-lock.json]: better to keep these at https if possible

**Comment by inodb** [comment]
@alisman Thanks! There's just a few http -> https changes in the package lock. Maybe just manually change those to https?

**Comment by adamabeshouse** [comment]
@inodb won't it keep getting reset every time we do npm install though?

**Comment by alisman** [comment]
@inodb i tried solution and it didn't work.  still getting the https -> http

**Pull request merged**